packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -14,24 +14,31 @@ import {
 import '@itwin/itwinui-css/css/tabs.css';
 import { Tab } from './Tab';
 
-type TabsOrientationProps =
+type TabsTypeProps =
```

Comment:
Isn't changing this prop name a breaking change?

Reply:
Good point. Since we are not exporting `TabsTypeProps`, I think this should be fine.

Just to be safe and confirm with other reviewers, @gretanausedaite, is this right?

Reply:
We are not using these Props outside this file so should be ok.